packages/atomic/src/utils/date-utils.ts

```diff
@@ -11,3 +11,44 @@ export function parseDate(date: ConfigType, format?: string) {
   }
   return parsedDate;
 }
+
+/**
+ * Transforms a timestamp to a date details object.
+ * @param timestamp Number
+ * @returns {year: number, month: string, dayOfWeek: string, day: number, hours: number,minutes: number}
+ */
+export function parseTimestampToDateDetails(timestamp: number) {
+  const daysOfWeek = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+  const months = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+
+  const date = new Date(timestamp);
```

Comment:
You should add some information on the `* @param timestamp Number` to indicate that you are expecting it to be an epoch number for example. Since you're removing the * 1000